src/guide/index.md

```diff
@@ -89,7 +89,7 @@ This example results reported a number of metric scores:
 
 ### Mutation Score Indicator (MSI)
 
-MSI is 47%. This means that 47% of all generated mutations were detected (i.e. kills, timeouts or fatal errors). The MSI is the primary Mutation Testing metric. Given the code coverage of 65%, there is a 18% difference so Code Coverage was a terrible quality measurement in this example.
+MSI is 47%. This means that 47% of all generated mutations were detected (i.e. kills, timeouts or fatal errors). The MSI is the primary Mutation Testing metric. Given the code coverage of 67%, there is a 20% difference so Code Coverage was a terrible quality measurement in this example.
```

Comment:
Code coverage !== Mutation Code Coverage, I'm not sure it was icorrect TBH, but let's make it less confusing.

Later, we will probably get rid of Mutation Code Coverage at all

Reply:
I can be that I'm wrong just confused me by reading it. Lets hope I didn't not confuse anyone with my change.